runners/CA/vaccinespotter.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
```

Comment:
I would like the ability to run the 3 stages of fetch, parse, and normalize separately, even if they are run at once for a normal production run.

Perhaps something like:
1. `runners/CA/vaccinespotter/fetch.py` that conditionally fetchs the url and uploads to S3.
2. `runners/CA/vaccinespotter/parse.py` that fetchs the raw blob stored in S3, parses it into a raw source record in ndjson format, and uploads that to S3
3.  `runners/CA/vaccinespotter/normalize.py` that fetchs the ndjson stored in S3, normalizes it to match the schema I proposed, and uploads it to S3.
4. `runners/CA/vaccinespotter/run.sh` that runs each runner in succession.